src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,6 @@
+# Your configuration properties
+quarkus.http.port=8080
```

Comment:
We should move this class to `src/main/resources` and name it `application.properties` now.

Reply:
ok